chains/bitlayer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "alias": "bitlayer",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://api.btrscan.com/scan/api"
+    },
+    "browserUrl": "https://www.btrscan.com/"
+  },
+  "id": "200901",
+  "name": "Bitlayer",
```

Comment:
Note to myself: It doesn't require any custom fee calculation schema. See https://docs.bitlayer.org/docs/BitlayerNetwork/AboutGas.